package.json

```diff
@@ -18,11 +18,9 @@
   "dependencies": {
     "ajv": "^6.5.5",
```

Comment:
Many of those dependencies don't appear to be actually use and could probably be removed. I didn't because I not familiar enough with this theme to know for sure what I'm testing for.